src/dataworker/DataworkerUtils.ts

```diff
@@ -343,7 +356,12 @@ export async function _buildPoolRebalanceRoot(
   maxL1TokenCountOverride: number | undefined,
   tokenTransferThreshold: BigNumberForToken,
   logger?: winston.Logger
-) {
+): Promise<{
+  runningBalances: RunningBalances;
+  realizedLpFees: RunningBalances;
+  leaves: PoolRebalanceLeaf[];
+  tree: MerkleTree<PoolRebalanceLeaf>;
+}> {
```

Comment:
I think this should be OK?

```suggestion
): Promise<PoolRebalanceRoot> {
```